discretize/base/base_tensor_mesh.py

```diff
@@ -417,12 +447,24 @@ def getInterpolationMat(self, loc, locType='CC', zerosOutside=False):
             M, the interpolation matrix
 
         """
-        return self._getInterpolationMat(loc, locType, zerosOutside)
+        if "locType" in kwargs:
+            warnings.warn(
+                "The locType keyword argument has been deprecated, please use loc_type. "
+                "This will be removed in discretize 1.0.0",
+                FutureWarning,
+            )
+            loc_type = kwargs["locType"]
+        if "zerosOutside" in kwargs:
+            warnings.warn(
+                "The zerosOutside keyword argument has been deprecated, please use loc_type. "
+                "This will be removed in discretize 1.0.0",
+                FutureWarning,
+            )
+            zeros_outside = kwargs["zerosOutside"]
+        return self._getInterpolationMat(loc, loc_type, zeros_outside)
 
-    def _fastInnerProduct(
-        self, projType, prop=None, invProp=False, invMat=False
-    ):
-        """ Fast version of getFaceInnerProduct.
+    def _fastInnerProduct(self, proj_type, prop=None, inv_prop=False, inv_mat=False):
```

Comment:
thinking through names here too. What do you think of 
- `projection_type`
- `inverse_property`
- `inverse_matrix`

Reply:
as this a hidden internal function, it doesn't make much of a difference. However, might as well change them internally to prop_type -> projection_type, prop->property, inv_prop -> invert_property, inv_mat -> invert_output

Reply:
or perhaps, `model` would be a better term than `property` as that is already a default python class